freqtrade/configuration/cli_options.py

```diff
@@ -273,6 +273,12 @@ def __init__(self, *args, **kwargs):
         type=check_int_positive,
         metavar='INT',
     ),
+    "download_trades": Arg(
+        '--dl-trades',
+        help='Download trades instead of OHLCV data. The bot will resample trades to the '
+             'desired timeframe as specified as --timeframes/-t.',
```

Comment:
It is noted ... ?

Reply:
Seems I was still sleep today's late morning... 🤣